simply_fit/simply_fit_api/tests.py

```diff
@@ -54,4 +54,17 @@ def test_addDateAndWorkoutHours(self):
 		self.assertEqual(None, apiUtil.getUser("parul"))
 		self.assertRaises(KeyError, apiUtil.getUserWorkOutHoursForDate, "parul", time)
 
+	'''
+	The purpose of this method is to test apiUtil.updateUser method.
+	The test includes creating a user, updating the users information, and then
+	confirming that updates were applied
+	'''
+	def test_updateUserInfo(self):
```

Comment:
Right now the function raises an attribute not found error. I will update the function to return a KeyError("User does not exist, update failed"), what do you think?